hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -1157,13 +1188,6 @@ private void finishActiveMasterInitialization(MonitoredTask status) throws IOExc
     }
 
     assignmentManager.checkIfShouldMoveSystemRegionAsync();
-    status.setStatus("Assign meta replicas");
-    MasterMetaBootstrap metaBootstrap = createMetaBootstrap();
-    try {
-      metaBootstrap.assignMetaReplicas();
-    } catch (IOException | KeeperException e){
-      LOG.error("Assigning meta replica failed: ", e);
-    }
```

Comment:
We don't need this anymore?

Reply:
Yes, in general we do not need this any more. All the works should be done in MTP. The only exception is that we still need to be compatible with the old way so we have a special logic when starting master, to schedule a MTP if the META_REPLICA_NUM has been changed.